src/sagemaker/environment_variables.py

```diff
@@ -46,6 +47,8 @@ def retrieve_default(
             retrieve the default environment variables. (Default: None).
         model_version (str): Optional. The version of the model for which to retrieve the
             default environment variables. (Default: None).
+        hub_arn (str): The arn of the SageMaker Hub for which to retrieve
```

Comment:
can you keep the docstring format for default args? so do `... model details from. (Default: None).`